src/main/scala/rocket/DCache.scala

```diff
@@ -503,4 +503,8 @@ class DCacheModule(outer: DCache) extends HellaCacheModule(outer) {
       flushing := false
     }
   }
+
+  // performance events
+  io.cpu.acquire := tl_out.a.fire()
+  io.cpu.release := tl_out.c.fire()
```

Comment:
As with NBDCache; you possibly want:
`io.cpu.acquire := edge.last(tl_out.a) && edge.opcode(tl_out.a) === TLMessages.Acquire`
`io.cpu.release := edge.last(tl_out.c)`